tomviz/CentralWidget.cxx

```diff
@@ -448,11 +453,56 @@ void CentralWidget::onDataSourceChanged()
   this->Internals->Timer.start();
 }
 
+//-----------------------------------------------------------------------------
 void CentralWidget::refreshHistogram()
 {
   this->setDataSource(this->ADataSource);
 }
 
+//-----------------------------------------------------------------------------
+void CentralWidget::onScalarOpacityFunctionChanged()
+{
+  pqApplicationCore* core = pqApplicationCore::instance();
+  pqServerManagerModel* smmodel = core->getServerManagerModel();
```

Comment:
Local variables so don't worry too much, but using camel case for follow on words helps (me at least), i.e. smModel pulls out the word model and I can infer what sm is and that it is an acronym more quickly.